### ISSUE TITLE
(MzIdentMLWriter) register AttributeError

### ISSUE THREAD

**Issue by lars-kolbowski** [mobiusklein/psims#19]
Hi Joshua,
I've been following the example from the [documentation](https://mobiusklein.github.io/psims/docs/build/html/mzid/examples.html) running
`writer.SearchDatabase.register("search_db_1")` or in fact any other `register()` throws:

`AttributeError: type object 'SearchDatabase' has no attribute 'kwargs'. Did you mean: 'args'?`

Stacktrace below:
```Traceback (most recent call last):
  File "/home/lars/work/repos/psims/test.py", line 34, in <module>
    writer.SearchDatabase.register("search_db_1")
  File "/home/lars/.local/share/virtualenvs/psims-PrEsMToZ/lib/python3.10/site-packages/psims/document.py", line 426, in register
    return self.kwargs['context'].register(self._func.__name__, identifier)
  File "/home/lars/.local/share/virtualenvs/psims-PrEsMToZ/lib/python3.10/site-packages/psims/document.py", line 412, in __getattr__
    return getattr(self._func, name)
AttributeError: type object 'SearchDatabase' has no attribute 'kwargs'. Did you mean: 'args'?```

**Comment by mobiusklein** [comment]
Thank you for reporting this. Apologies, I'll get that fixed shortly.

**Comment by mobiusklein** [comment]
This should be addressed in v1.2.6 which is being released to PyPI momentarily

**Comment by lars-kolbowski** [comment]
Thank you!
